tests/prepare/ansible/test.sh

```diff
@@ -38,9 +41,9 @@ rlJournalStart
 
                 # Run given method
                 if [ "$PROVISION_HOW" = "local" ]; then
-                    rlRun "tmt run -i $run --scratch -av provision -h $PROVISION_HOW           plan -n /$plan"
+                    rlRun "tmt run -i $run --scratch -av provision -h $PROVISION_HOW           plan -n /$plan -d"
```

Comment:
Remove here as well? Btw, I think, `-d` does not have any effect here anyway.

Reply:
done